### PR TITLE
Mentorsubject and config

### PR DESCRIPTION
**Pull request by beatthat**
adds a gql 'config' query

### CODE REVIEW

File: node/src/gql/types/config.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+} from 'graphql';
+
+export const ConfigType = new GraphQLObjectType({
+  name: 'Config',
+  fields: () => ({
+    cmi5Enabled: { type: GraphQLBoolean },
+    cmi5Endpoint: { type: GraphQLString },
+    cmi5Fetch: { type: GraphQLString },
+    mentorsDefault: { type: GraphQLList(GraphQLString) },
+    styleHeaderLogo: { type: GraphQLString },
```
Comment (kycarr) [inline node/src/gql/types/config.ts:21]: Will add the config settings from Dan's branch here in a different PR?

File: node/src/gql/query/index.ts [hunk 2]
```diff
@@ -5,19 +5,21 @@ Permission to use, copy, modify, and distribute this software and its documentat
 The full terms of this copyright and license should always be found in the root directory of this software deliverable as "license.txt" and if these terms are not found with this software, please contact the USC Stevens Center for the full license.
 */
 import { GraphQLObjectType } from 'graphql';
+import config from './config';
```
Comment (kycarr) [inline node/src/gql/query/index.ts:8]: I assume we'll also need an updateConfig mutation eventually so someone can edit the config settings in admin? I assume we'll want to limit that to admin users only, something to consider for the future.

**Review by kycarr — APPROVED** [review]

File: node/src/gql/types/config.ts (hunk 1 above)
Comment (beatthat) [reply node/src/gql/types/config.ts:21]: Yeah, this is enough to make the updated static client work. Once this is in place, Dan can add whatever additional config settings he needs

File: node/src/gql/query/index.ts (hunk 2 above)
Comment (beatthat) [reply node/src/gql/query/index.ts:8]: Yes. This just assumes the config is set, but we should have another story where User can edit config from /admin app

**Pull request merged**